ONWServices/Models/Game.cs

```diff
@@ -2,14 +2,42 @@
 using MongoDB.Bson.Serialization.Attributes;
 using ONWServices.Models.Serializers;
 using System;
+using System.Collections.Generic;
 
 namespace ONWServices.Models
 {
     [BsonIgnoreExtraElements]
     public class Game : BaseDocument
     {
+        public Game() {
+            Status = GameStatus.New;
+            SelectedRoles = new List<Role>
+            {
+                Role.Warewolf,
+                Role.Seer,
+                Role.Robber,
+                Role.Troublemaker
+            };
+        }
+
         [BsonElement("gameId")]
         [BsonSerializer(typeof(GuidStringSerializer))]
         public Guid GameId { get; set; }
+
+        public GameStatus Status { get; set; }
+        public List<Role> SelectedRoles { get; set; }
+    }
+
+    public enum GameStatus
+    {
+        New
+    }
+
+    public enum Role
```

Comment:
Maybe `GameRole`/`UserRole`/`PlayerRole`? Also, perhaps this one should be in its own file since it will be used by both the `Game` class here and the `User`/`Player` class in the future (i.e. it's not "owned" by `Game`).